CorsixTH/Lua/dialogs/fullscreen/staff_management.lua

```diff
@@ -336,7 +336,8 @@ function UIStaffManagement:draw(canvas, x, y)
   self.portrait_back.visible = false
   -- If a staff member is selected, draw picture, skill etc
   if self.selected_staff then
-    local profile = self.staff_members[self.category][self.selected_staff].profile
+    local staff = self.staff_members[self.category][self.selected_staff]
```

Comment:
This isn't you, but if you wanted to help the code a little, catching the do not proceed action would result in a lot less indenting.
```lua
if not self.selected_staff then
  self.ui.playSound("wrong2.wav")
  return
end

and then everything in the current condition goes left and the reader doesn't have to look very far to see that we fail and return early when there's no selected_staff and the meat of the method applies so selected_staff.

Reply:
Actually, this is the draw method. What in the world are we doing playing sounds from the draw method?